extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/util/PropertyUtils.java

```diff
@@ -41,8 +46,13 @@ public static State getState(ChannelUID channelUID, AstroChannelConfig config, O
         if (value == null) {
             return UnDefType.UNDEF;
         } else if (value instanceof Calendar) {
+            // TODO use LocationProvider service once it's merged https://github.com/eclipse/smarthome/issues/3936
+            ZoneId zone = ZoneId.of("+03:00");
```

Comment:
This is still an issue, right? This arbitrary time zone must not be used in production I guess.

Reply:
Indeed, this works nicely only if you are living in the EEST time zone. In Summer. 😉

Reply:
Of course it must not! I am waiting for the other PR to be merged and I will do this with the i18nProvider where I added time zone delivery.

Reply:
You mean #4256, right? Just rebase it an while you're at it maybe incorporate my suggestions (or not - nothing serious). It fully makes sense to integrate it directly, good plan.